include/mbedtls/ssl_cache.h

```diff
@@ -104,7 +104,8 @@ void mbedtls_ssl_cache_init(mbedtls_ssl_cache_context *cache);
  *                        associated with \p session_id, if present.
  *
  * \return                \c 0 on success.
- * \return                A negative error code on failure.
+ * \return                #MBEDTLS_ERR_SSL_CACHE_ENTRY_NOT_FOUND if there is
```

Comment:
Please re-add a line for other error codes, since other errors are possible (e.g. out of memory). We don't need to list them all since they're likely to be errors that the caller wouldn't want to handle specifically, but we should list the possibility of other errors.